config/swagger.yaml

```diff
@@ -1163,6 +1219,7 @@ paths:
     get:
       tags:
         - external
+        - debug
```

Comment:
I see this is still not dependent on enable_debug_endpoints. Ok.

Reply:
Yes, will be a subject to another PR

Reply:
I am not sure `/version` shall be put in same bucket `debug` as `/balances` because much lighter. Though can be refined later?